js/main.js

```diff
@@ -0,0 +1,37 @@
+// Первая функция
+// Источник: https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+// Подраздел "Получение случайного целого числа в заданном интервале, включительно"
+
+function getRandomIntInclusive(min, max) {
+  // "А также придумайте, как функция должна вести себя,
+  // если передать значение «до» меньшее, чем значение «от», или равное ему"
+  // Не знаю как правильно надо было сделать, пока сделала принудительный перевод в единицу
+  if (min <= 0) {
+    min = 1;
+  }
+  if (max <= 0) {
+    max = 1;
+  }
```

Comment:
```suggestion
  if (min <= 0) {
    throw new Error(`Договаривались min меньше нуля не получать, а пришло ${min}`);
  }
  if (max <= 0) {
    throw new Error(`Доваривались max меньше нуля не получить, а пришло $(max)`);
  }
  if(min>max){
    throw new Error(`договаривались, что min<max, а получено ${min}?${max}`);
  }
```

смысл в том, что пользователь- программист, позвавший функцию, договоренность читал и знает. Но.... раз пришло что-то не то, возможно у него какая-то глубинная ошибка есть и тут он ее быстрее найдет, если сразу сообщить.